### PR TITLE
display duration in years and months if it exceeds 12 months

### PR DESCRIPTION
**Pull request by giom-l**
Display missions duration in years and months if it exceeds 12 months

### CODE REVIEW

File: src/components/missions/Mission.vue [hunk 1]
```diff
@@ -5,7 +5,8 @@
       <img class="mission__client-image" :src="mission.client.image.src" :alt="mission.client.image.alt">
       <div class="mission__header">
         <div class="mission__client-name">{{mission.client.name}}</div>
-        <div class="mission__duration">{{ $tc('month', duration) }}</div>
+        <!-- <div class="mission__duration">{{ $tc('month', duration) }}</div> -->
```
Comment (blacroix) [inline src/components/missions/Mission.vue:8]:
```suggestion
```

File: src/components/missions/Mission.vue [hunk 2]
```diff
@@ -39,7 +40,9 @@
         if (this.mission.endDate) {
           endDate = this.mission.endDate;
         }
-        return dayjs(endDate).diff(this.mission.beginDate, 'month');
+        let years = dayjs(endDate).diff(this.mission.beginDate, 'year')
+        let monthes = dayjs(endDate).diff(dayjs(this.mission.beginDate).add(years, 'year'), 'month')
```
Comment (blacroix) [inline src/components/missions/Mission.vue:44]: Could we rename `monthes` to `months`?

Comment (giom-l) [reply src/components/missions/Mission.vue]: done

File: src/components/missions/Mission.vue (hunk 1 above)
Comment (giom-l) [reply src/components/missions/Mission.vue]: done

**Pull request merged**